pages/search/_query/index.vue

```diff
@@ -1,31 +1,11 @@
 <template>
-	<b-row align-h="center">
-		<div id="container">
-			<h1 id="title">
-				<b>SEARCH: "{{ searchQuery }}"</b>
-			</h1>
-			<p>We found {{ pagesPosts.length }} results for your search.</p>
-			<b-row v-for="row in groupedPosts" :key="row.id">
-				<b-col id="api-content" v-for="x in row" :key="x.id">
-					<Post
-						v-if="row.length>1"
-						:picture="x.picture"
-						:title="x.title"
-						:date="x.date"
-						:slug="x.slug"
-					/>
-					<Post
-						v-else
-						:picture="x.picture"
-						:title="x.title"
-						:date="x.date"
-						:slug="x.slug"
-						style="width: 50%"
-					/>
-				</b-col>
-			</b-row>
-		</div>
-	</b-row>
+	<div class="container">
+		<h1 id="title">
+			SEARCH: "{{ searchQuery }}"
+		</h1>
+		<p>We found {{ pagesPosts.length }} results for your search.</p>
+		<NewsGrid :postList="filterdBlogs" />
```

Comment:
A couple comments: `filterdBlogs` has a typo in it. Also, you've searched both posts and pages below but you're only passing the post results to the NewsGrid component here.

Reply:
in the original we count site both pages and posts showed up in search and only posts in tags